src/Sentry.Unity.Editor/SentryAssetPostprocessor.cs

```diff
@@ -0,0 +1,52 @@
+using System.Linq;
+using System.Threading;
+using UnityEditor;
+using UnityEditor.PackageManager;
+using UnityEngine;
+
+namespace Sentry.Unity.Editor
+{
+    public class SentryAssetPostprocessor : AssetPostprocessor
+    {
+        private static void OnPostprocessAllAssets(string[] importedAssets, string[] deletedAssets, string[] movedAssets, string[] movedFromAssetPaths)
+        {
+            var imported = importedAssets.Any(path => path.StartsWith("Packages/"));
+            var deleted = deletedAssets.Any(path => path.StartsWith("Packages/"));
+
+            if (imported || deleted)
+            {
+                InitializeOnLoad();
+            }
+        }
+
+        /**
+         * Instead of hooking into AssetPostprocessing we could InitializeOnLoad
+         * But this gets called on domain reload (i.e. enter playmode) and could be disabled by the user.
+         */
+        // [InitializeOnLoadMethod]
+        private static void InitializeOnLoad()
+        {
+            // ASYNC! Fetching all packages the current project depends on
+            var listRequest = Client.List(true);
+            while (!listRequest.IsCompleted)
+            {
+                Thread.Sleep(100);
```

Comment:
This might freeze up the editor though, right?
Can we schedule the continuation to run async too? If this was TPL it would be `listRequest.ContinuesWith(p => ..` but I'm not sure the type here (need to load the editor to do a proper review).